includes/admin/class-wp-job-manager-admin.php

```diff
@@ -89,7 +89,7 @@ public function admin_enqueue_scripts() {
 		WP_Job_Manager::register_select2_assets();
 
 		$screen = get_current_screen();
-		if ( in_array( $screen->id, apply_filters( 'job_manager_admin_screen_ids', [ 'edit-job_listing', 'plugins', 'job_listing', 'job_listing_page_job-manager-settings', 'job_listing_page_job-manager-addons' ] ), true ) ) {
+		if ( in_array( $screen->id, apply_filters( 'job_manager_admin_screen_ids', [ 'edit-job_listing', 'plugins', 'job_listing', 'job_listing_page_job-manager-settings', 'job_listing_page_job-manager-addons', 'edit-job_listing_type', 'dashboard' ] ), true ) ) {
```

Comment:
Is this needed?

Reply:
With the notices as they currently display, if this isn't added, then navigating to **Job Listings --> Job Types** or WP Admin home shows the notice as so:

![styles](https://user-images.githubusercontent.com/2500940/232081594-464a9c27-c410-467a-a3d5-6dd40289081c.png)

Did we want to hide the notices on these pages instead?

Reply:
About this, I'd say that it would be nice to add only the needed assets, otherwise, we add a lot of unneeded assets to the page.

But I see that it's the current standard here and it seems to be including all the assets used in the admin. And since it's in the admin, I think it's fine to keep that for now to avoid a big refactor.

Reply:
Thanks for the explanation @anaemnesis ! I am good with this approach too